package.json

```diff
@@ -98,5 +98,8 @@
     "*.json": "prettier --write",
     "*.yml": "prettier --write",
     "*.md": "prettier --write"
+  },
+  "resolutions": {
+    "tough-cookie": "4.1.3"
```

Comment:
if we do this we should create GH issue to clean this up when verdaccio ships patch.
Or, maybe better, since this is medium severity and only used in our build time we could ignore this for now.
Can you check if this problem affects us in any meaningful way ?

Reply:
Right, we only use it in verdaccio, which doesn't affact anything other than build/dev. 

```
└─┬ verdaccio@5.27.0
  └─┬ request@2.88.2
    └── tough-cookie@2.5.0
```

I think it makes sense to dismiss the alert.